jinja2/utils.py

```diff
@@ -506,12 +506,14 @@ def current(self):
         """Returns the current item."""
         return self.items[self.pos]
 
-    def __next__(self):
+    def next(self):
```

Comment:
I just merged https://github.com/pallets/jinja/pull/470 which affects some of the next behavior in `_compat.py`. I haven't looked closely enough at this PR to understand if it was affected, just noticed you mentioned `next`/`__next__` and thought I'd mention the merge.